README.md

```diff
@@ -57,12 +57,38 @@ This project supports a basic [Vagrant](http://docs.vagrantup.com/v2/getting-sta
 
 1. Run vagrant up command
 
-    vagrant up
+        vagrant up
```

Comment:
It was not correctly indented (and it didn't appear as code). I follow this: https://help.github.com/articles/github-flavored-markdown/#fenced-code-blocks (at the end it says: *"Keep in mind that, within lists, you must indent non-fenced code blocks eight spaces to render them properly."*)